website/docs/components/time/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Time
+description: The Time component is used to display dates and time in a consistent format.
+caption: Displays dates and times in a consistent format.
+related: ['components/text']
+links:
+  figma: {Link}
+  github: https://github.com/hashicorp/design-system/tree/main/packages/components/src/components/hds/time
+previewImage: assets/illustrations/components/time.jpg
+navigation:
+  keywords: ['time']
```

Comment:
Should the status be added here with the version we expect this to be released in?

Reply:
Added, thanks